src/Process/Runner/MutationTestingRunner.php

```diff
@@ -73,6 +75,7 @@ class MutationTestingRunner
     public function __construct(
         MutantProcessFactory $processFactory,
         MutantFactory $mutantFactory,
+        MutantFiltersHandler $pluginsHandler,
```

Comment:
```suggestion
        MutantFiltersHandler $mutantFilterer,
```

In a sense we don't care where it comes from at this point, plugin or elsewhere. I'm even wondering if the two `filter()` calls should be moved to such filterers